### DISCUSSION TITLE
Ability to find and remove blank pages ?

### DISCUSSION THREAD

**Discussion by nodecentral** [Stirling-Tools/Stirling-PDF#133] [Q&A]
Hi

I couldn’t see it explicitly called out, is there the ability to scan a pdf and remove any blank pages ?

**Comment by Frooodle** [comment]
No this is not currently functioning
It can only delete exact pages either by UI or page number no automatic detection for blank pages 
 
If you raise a github issue ticket as a feature request I will look into adding it !

**Comment by nodecentral** [comment]
Done, thanks @Frooodle
